Sources/Swiftly/Init.swift

```diff
@@ -67,21 +67,21 @@ struct Init: SwiftlyCommand {
         // Give the user the prompt and the choice to abort at this point.
         if !assumeYes {
 #if os(Linux)
-            let sigMsg = " In the process of installing the new toolchain swiftly will add swift.org GnuPG keys into your keychain to verify the integrity of the downloads."
+            let sigMsg = " In the process of installing the new toolchain, swiftly will add swift.org GnuPG keys into your keychain to verify the integrity of the downloads."
 #else
             let sigMsg = ""
 #endif
             let installMsg = if !skipInstall {
-                "\nOnce swiftly is installed it will install the latest available swift toolchain.\(sigMsg)\n"
+                "\nOnce swiftly is installed it will install the latest available Swift toolchain.\(sigMsg)\n"
             } else { "" }
 
             SwiftlyCore.print("""
-            Swiftly will be installed into the following locations:
+            Swiftly will install files into the following locations:
```

Comment:
```suggestion
            Swiftly installs files into the following locations:
```
(removes the future tense, which makes the action more direct)